GitCommands/Settings/AppSettings.cs

```diff
@@ -1603,7 +1606,16 @@ public static void SaveSettings()
             {
                 SettingsContainer.LockedAction(() =>
                 {
-                    SettingsContainer.Save();
+                    _globalMutex ??= new Mutex(initiallyOwned: false, name: SettingsFilePath.ToPosixPath());
```

Comment:
should this be shared?
https://docs.microsoft.com/en-us/dotnet/api/system.threading.mutex.openexisting?view=net-6.0#system-threading-mutex-openexisting(system-string)

Reply:
Yes, that's the trick. GE instances must write the settings one after the other. (The last one wins. For this I leave #8006 open.)

Reply:
Why not use OpenExisting in this PR?
(This PR is probably an improvement anyway.)

Reply:
The use of `OpenExisting` needs a lot of additional code for little benefit (protection by access rights).
From https://docs.microsoft.com/en-us/dotnet/api/system.threading.mutex.-ctor?view=net-6.0#system-threading-mutex-ctor(system-boolean-system-string):
> If a name is provided and a synchronization object of the requested type already exists in the namespace, the existing synchronization object is used.

Reply:
A named mutex is always shared between processes - at least in the same session.

> If a synchronization object of a different type already exists in the namespace, a WaitHandleCannotBeOpenedException is thrown.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.threading.mutex.-ctor?view=net-6.0#system-threading-mutex-ctor(system-boolean-system-string)
```
The name may be prefixed with Global\ or Local\ to specify a namespace. When the Global namespace is specified, the synchronization object may be shared with any processes on the system. When the Local namespace is specified, which is also the default when no namespace is specified, the synchronization object may be shared with processes in the same session. On Windows, a session is a login session, and services typically run in a different non-interactive session. On Unix-like operating systems, each shell has its own session. 
```

So it seems to be working in Windows, append `Global` to prepare for other platforms.

Have you tested multiple closing (by adding a delay or so).

Reply:
OK, I admit I stopped reading after "On Windows, a session is a login session".
Adding `"Global\"`.

> Have you tested multiple closing (by adding a delay or so).

Yes, it takes 30s to close three instances at the same each waiting 10s - both with and without `"Global\"`